### PR TITLE
feat: Add Zabbix credential only node

### PR DESCRIPTION
**Pull request by adina-hub**
## Review / Merge checklist

- [ ] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

**Comment by Joffcom** [comment]
Hey @adina-hub,

Thanks for the PR, We have created "GHC-371" as the internal reference to get this reviewed.

One of us will be in touch if there are any changes needed, in most cases this is normally within a couple of weeks but it depends on the current workload of the team.

File: packages/nodes-base/credentials/ZabbixApi.credentials.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+	authenticate: IAuthenticateGeneric = {
+		type: 'generic',
+		properties: {
+			headers: {
+				Authorization: '=Bearer {{$credentials.apiToken}}',
+			},
+		},
+	};
+
+	test: ICredentialTestRequest = {
```
Comment (Joffcom) [inline packages/nodes-base/credentials/ZabbixApi.credentials.ts:50]: This returns a 200 status code even if the API token is invalid, You need to check the message body using the rules

**Review by Joffcom — COMMENTED** [review]
Left some notes, As the `ZabbixAuthApi` is using a deprecated method we should probably skip it and only move forward with the API token optoin.

**Comment by Joffcom** [comment]
@adina-hub the credential test still returns a success if the toke is incorrect or missing. Everything else looks good though, I have been testing against a self hosted Zabbix instance and Zabbix Cloud.

**Comment by adina-hub** [comment]
@Joffcom Last changes are pushed, credentials were tested again, now it works.

**Comment by Joffcom** [comment]
@adina-hub that is perfect thanks 🙏🏻

**Review by Joffcom — APPROVED** [review]
This looks good thanks.

**Comment by janober** [comment]
Got released with [`n8n@1.70.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.70.0)

**Pull request merged**